spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -136,5 +135,18 @@
 				</pluginRepository>
 			</pluginRepositories>
 		</profile>
+		<profile>
+			<id>test</id>
```

Comment:
This is required to get Spring core 5.0.0.BUILD-SNAPSHOT, so that we can set up the auto configuration unit test environments, inspired on how Boot does it currently.